<ui version="4.0" >
 <class>CoreConnectDlg</class>
 <widget class="QDialog" name="CoreConnectDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Select Core Account</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Available Quassel Core accounts:</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QListWidget" name="accountList" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="newAccount" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="delAccount" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../images/icons.qrc" >:/default/edit_remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="editAccount" >
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../../images/icons.qrc" >:/default/edit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="doConnect" >
     <property name="text" >
      <string>Connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>accountList</tabstop>
  <tabstop>doConnect</tabstop>
  <tabstop>newAccount</tabstop>
  <tabstop>delAccount</tabstop>
  <tabstop>editAccount</tabstop>
  <tabstop>delAccount</tabstop>
  <tabstop>editAccount</tabstop>
  <tabstop>accountList</tabstop>
  <tabstop>newAccount</tabstop>
 </tabstops>
 <resources>
  <include location="../../images/icons.qrc" />
 </resources>
 <connections/>
</ui>
